src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacScope.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2006, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2006, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
copyright in this (and other files need to be updated)

Reply:
Fixed.